ccpage135_141.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Write a function called display_message() that prints one sentence
+# telling everyone what you are learning about in this chapter.
+def display_message():
+    """Share what I'm learning about in this chapter"""
+    print("I'm learning about functions.")
+
+display_message()
+
+
+# Write a function called favorite_book() that accepts one parameter, title.
+# The function should print a message.
+def favorite_book(title):
+    """Tell my favorite book"""
+    print(f"My favorite book is {title.title()}.")
+
+favorite_book('curious george')
+
+
+# Write a function called make_shirt() that accepts a size and the text of a message that should be printed on the shirt.
+# The function should print a sentence summarizing the size of the shirt.
+# Call the function once using positional arguments to make a shirt.
+# Call the function a second time using keyword arguments.
+
+def make_shirt(size, text):
+    """Describe t-shirt"""
+    print(f"The t-shirt is size {size} and says \'{text}.\'")
+
+make_shirt('small', 'i love python')
+make_shirt(text = 'hello world', size = 'medium')
+
+
+# Modify the make_shirt() function so that shirts are large by default with a message that reads I love Python.
+# Make a large shirt and a medium shirt with the default message, and a shirt of any size with a different message.
+def make_shirt(text = 'i love python', size = 'large'):
```

Comment:
Same thing here regarding no need for spaces around the equals sign on default parameters.